tensorflow/python/user_ops/user_ops.py

```diff
@@ -26,3 +26,11 @@
 def my_fact():
   """Example of overriding the generated code for an Op."""
   return gen_user_ops._fact()
+
+@ops.RegisterGradient("Cholesky")
```

Comment:
This should go in tensorflow/python/ops/linalg_grad.py. You might look at how ReluGrad is registered in nn_grad.py as an example.